src/solvers/floatbv/float_bv.h

```diff
@@ -18,14 +18,6 @@ Author: Daniel Kroening, kroening@kroening.com
 class float_bvt
```

Comment:
mark the class as final?

Reply:
I contemplated that, but it wasn't entirely clear whether there is value in this? What would the desired benefit be? (FWIW, I don't think deriving from this class would strictly be wrong.)